src/Functional.CQS.AOP.CommonTestInfrastructure/Functional.CQS.AOP.CommonTestInfrastructure.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
```

Comment:
For all `.csproj` files, it may be useful to enable `Treat All Warnings As Errors`, for both `Debug` _and_ `Release` configurations.

Reply:
There are currently two "`licenseUrl` element will be deprecated" warnings because of how I currently have NuGet package configurations set up.  I'll fix this as part of a different PR.